src/components/EditButton/styles.module.scss

```
@@ -0,0 +1,51 @@
+    font-size: 14px;
+    font-weight: 500;
+    letter-spacing: 0.1px;
+    line-height: 20px;
+    text-transform: none !important;
+    transition: background 160ms ease, color 160ms ease, transform 160ms ease;
+
+    svg {
+        width: 20px;
+        height: 20px;
+        flex: 0 0 auto;
+        color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)